hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -753,5 +817,58 @@
        }
       ]
     },
+    {
+      name: "ERR_CODE",
+      desc: "Hardware detection of error conditions status register",
+      swaccess: "rw",
```

Comment:
it might be good to make this `rw1c`, that way it behaves like interrupts and you don't accidentally wipe out a bit during a read modified write to clear.

Reply:
Steve and I have discussed this a bit more. The fifo_err interrupt is a rw1c bit in the interrupt register. This register is just the supporting status for that interrupt. The idea is that is would be read as a whole value to get the whole status, so there is no danger of losing bit status since this register should not be treated on a bit basis. Given the type of error (casters up), I doubt this register will ever need to be cleaned up, but it can be written with a zero. If you don't like this explanation, it certainly can be changed, but it would be in 3 places (all of the entropy complex blocks) for consistency.